src/azul/service/elasticsearch_service.py

```diff
@@ -532,9 +532,10 @@ def _generate_paging_dict(self,
                                         search_after=search_after)
 
         def page_link(*, previous):
-            return pagination.link(previous=previous,
-                                   catalog=catalog,
-                                   filters=json.dumps(filters))
+            url = pagination.link(previous=previous,
+                                  catalog=catalog,
+                                  filters=json.dumps(filters))
+            return str(url) if url else None
```

Comment:
Conflates `None` with other false-ish values.

Reply:
would `return str(url) if url is not None else None` be a better alternative?

Reply:
Not really.

```suggestion
            return None if url is None else str(url)
```